crates/cheatnet/src/runtime_extensions/forge_runtime_extension/file_operations.rs

```diff
@@ -124,98 +114,32 @@ mod tests {
             "ad": "string",
             "test": ["1",2,"3",4]
         }"#;
-        let result = json_values_sorted_by_keys(string).unwrap();
-        let expected_result = ["\"string\"", "4", "\"1\"", "2", "\"3\"", "4"];
+        let (_temp, file_path) = create_file(string);
+        let result = read_json(file_path).unwrap();
+        let mut expected_result = "string".serialize_as_felt252_vec();
+        expected_result.push(Felt252::from(4)); // .test.len()
```

Comment:
```suggestion
        expected_result.push(Felt252::from(4));
```

nitpick, I don't think this is necessary but if you disagree, that's fine. In that case we might want to improve on this comment.